cpp/modules/math.gl/core/src/core.h

```diff
@@ -251,14 +251,29 @@ Vector4<coord>::Vector4(const Vector4<othercoord> &other)
       w{static_cast<coord>(other.w)} {}
 
 template <typename coord>
-Vector4<coord> vectorProduct(const Vector4<coord> &, const Vector4<coord> &);
+Vector4<coord> vectorProduct(const Vector4<coord> &, const Vector4<coord> &, const Vector4<coord> &);
 
 template <typename coord>
 Vector4<coord> elementProduct(const Vector4<coord> &, const Vector4<coord> &);
 
 template <typename coord>
-auto Vector4<coord>::toVector3() const -> Vector3<coord> {
-  return Vector3<coord>(x, y, z);
+auto operator+(const Vector4<coord> &v1, const Vector4<coord> &v2) -> Vector4<coord> {
+  return Vector4<coord>(v1.x + v2.x, v1.y + v2.y, v1.z + v2.z, v1.w + v2.w);
```

Comment:
Initializers should use curly brace syntax, same applies to the rest of the added return values

```suggestion
  return Vector4<coord>{v1.x + v2.x, v1.y + v2.y, v1.z + v2.z, v1.w + v2.w};
```

Reply:
+1